### DISCUSSION TITLE
SP-API Product Types and Listing Inventory

### DISCUSSION THREAD

**Discussion by gletech** [amzn/selling-partner-api-models#3168] [General]
Hi

We are currently using MWS to support 60+ clients that all list in gated categories Sports and Entertainment Collectibles and Collectible Coins

There are no Product Types for these as far as we can tell, nor are they in the update coming out next in October

Is there a way to use the ListingsApi without a Product Type? We can take at least a guess at the field names with our current flatfile upload. 

Thanks
Mike

**Comment by vaidas-lungis** [comment]
There is a product type `PRODUCT`
You could try with it

**Reply by trycatchsoft** [reply]
when we list a product with asin code, can we just write "product" on product type?

**Reply by vaidas-lungis** [reply]
Looks like it. This is what [Listings Items API Use Case Guide](https://github.com/amzn/selling-partner-api-docs/blob/main/guides/en-US/use-case-guides/listings-items-api-use-case-guide/listings-items-api-use-case-guide_2020-09-01.md) states under **Considerations** section

> Partially Supported Product Types: For Amazon product types not yet fully supported by the Listings Items API, offer-only submissions for existing ASINs and partial updates are supported by using the PRODUCT product type.
